src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft.Win32.SystemEvents.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
   <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
   <PropertyGroup>
+    <DefineConstants Condition="$(TargetFramework.StartsWith('netstandard2.0')) or $(TargetFramework.StartsWith('netcoreapp2.0')) or $(TargetFramework.StartsWith('netcoreapp3.0'))">$(DefineConstants);FEATURE_CER</DefineConstants>
```

Comment:
The CER APIs were no-ops in netcoreapp2.0 and necoreapp3.0 as well. It is not necessary to define `FEATURE_CER` for those.

Reply:
Thanks -- removing it everywhere.  Also seeing that FEATURE_GAC was not needed either.

Reply:
It looks like the only place `FEATURE_CER` is appropriate is in `System.Reflection.Metadata`, where it targets `netstandard1.1` and `netstandard2.0` and compiles the source files for those targets.

Reply:
And same is true for `FEATURE_GAC` -- `System.Reflection.Metadata` looks like the only necessary place for it.